src/providers/bitski-provider.ts

```diff
@@ -182,7 +184,7 @@ export class BitskiProvider extends OAuthHttpProvider {
 
         const ethSendTransactionUrl = `${BITSKI_WEB_HOST}/eth-send-transaction`;
         const encodedPayload = btoa(JSON.stringify(payload));
-        const txnParams = `network=${this.networkName}&payload=${encodedPayload}&accessToken=${user.access_token}`;
+        const txnParams = `network=${this.networkName}&payload=${encodedPayload}&accessToken=${accessToken}`;
```

Comment:
Can we switch this to `referrerAccessToken` to disambiguate in web UI?

Reply:
Updated this